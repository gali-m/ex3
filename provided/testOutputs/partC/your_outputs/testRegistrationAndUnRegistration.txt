an exception has occurred
EventDoesNotExist
an exception has occurred
EventDoesNotExist
EventDoesNotExist
an exception has occurred
an open event 2/1/2000

an open event 5/1/2000

an open event 8/1/2000

an open event 11/1/2000

an open event 14/1/2000

