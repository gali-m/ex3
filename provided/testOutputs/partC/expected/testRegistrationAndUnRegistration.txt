an exception has occurred
EventDoesNotExist
InvalidStudent
EventDoesNotExist
EventDoesNotExist
InvalidStudent
an open event 2/1/2000

an open event 5/1/2000

an open event 8/1/2000

an open event 11/1/2000

an open event 14/1/2000

